src/open_inwoner/cms/tests/test_middleware.py

```diff
@@ -0,0 +1,70 @@
+from django.conf import settings
+from django.test import TestCase, override_settings
+from django.urls import reverse
+
+from pyquery import PyQuery as pq
```

Comment:
I originally used Webtest and it's pyquery extension, but to get the 2factor test setup working I had to use mixin code which depends on Django's regular TestCase/client, so I had to do my own pyqueerying.